scripts/hudson/narayana.sh

```diff
@@ -316,6 +316,17 @@ function qa_tests {
   [ $ok1 = 0 -a $ok2 = 0 ] || fatal "some qa tests failed"
 }
 
+# when building on JDK 7 we get the error
+#   "Sonatype no longer supports TLSv1.1 and below (effective, June 18th, 2018).
+# There are two fixes: upgrade the JDK or use -Dhttps.protocols=TLSv1.2 when starting java
+JAVA_VER=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;')
+if [ "$JAVA_VER" -lt 18 ]; then
+  echo "JDK version < 18 - setting -Dhttps.protocols=TLSv1.2"
+  AS_XARGS="-Dhttps.protocols=TLSv1.2"
+else
+  AS_XARGS="-Dhttps.protocols=TLSv1.2"
```

Comment:
I forgot to delete this line.

Reply:
Thanks Mike, I have pushed a change to remove the else clause and the line